presets/react/rollup.config.js

```diff
@@ -55,7 +57,9 @@ export default (pkg, pkgPath) => ({
     peerDepsExternal(),
     postcss({
       extract: true,
-      modules: true,
+      modules: {
+        globalModulePaths: [/mapbox-gl/],
```

Comment:
Postcss plugin doesn't have `exclude` settings like other plugins do. So I think using this should be okay.

Should we process any file from node_modules with postcss? If yes, then we have to move these settings to map's own rollup config. If not, then `/node_modules/` should be ok.

Reply:
i think we have to assume same as with javascript, that node_modules includes precompiled CSS.